.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: generic
-sudo: false
-dist: trusty
+dist: bionic
```

Comment:
Trusty is 14.04 (6 year old LTS)
Xenial is 16.04 (4 year old LTS)
Bionic is 18.04 (2 year old LTS --- seems most reasonable to use)

`sudo: false` is a setting that is now deprecated and does nothing.

Reply:
We will need to eventually migrate this repo to github actions to unify with bokeh.